README.md

```diff
@@ -9,6 +9,9 @@ ZHA device handlers bridge the functionality gap created when manufacturers devi
 
 Custom quirks implementations for zigpy implemented as ZHA Device Handlers are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://stdavedemo.readthedocs.io/en/latest/device-type-developers-guide/) as well that of [Zigbee-Herdsman Converters (formerly Zigbee-Shepherd Converters) as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html), meaning they are virtual representation of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms. See [Device Specifics](#Device-Specifics) for details.
 
+
+For supporting Tuya devices, see [using the TuyaQuirkBuilder](/tuya.md).
```

Comment:
Very minor, but just realized this link isn't clickable on PyPi now, as it leads to https://pypi.org/tuya.md then.
We may want to use the full link here? Or at least see what (our) other repos do. Probably doesn't really matter though 😄

Reply:
Didn't even consider PyPi when I did that. Once we get this latest batch of PRs through, I need to add the new convenience methods to the docs and will change this to the full link on that PR.

Reply:
Yeah, I don't think anybody reads the docs there, but.. 😄